### PR TITLE
use requested path to configs

### PR DESCRIPTION
**Pull request by jesopo**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -170,7 +170,10 @@ const defaultConfig: IConfig = {
 };
 
 export function read(): IConfig {
-    const content = fs.readFileSync(`./config/${process.env.NODE_ENV || 'default'}.yaml`, "utf8");
+    const config_dir = process.env.NODE_CONFIG_DIR || "./config";
+    const config_file = `${process.env.NODE_ENV || "default"}.yaml`
+
+    const content = fs.readFileSync(`${config_dir}/${config_file}`, "utf8");
```
Comment (Gnuxie) [inline src/config.ts:176]: Should we use [path](https://nodejs.org/api/path.html) parse and join just to be on the safe side here?

Comment (jesopo) [reply src/config.ts:176]: `join` yes, but use `parse` for what?

Comment (Gnuxie) [reply src/config.ts:176]: ahh right, i forgot that paths are just strings sorry

**Review by Gnuxie — APPROVED** [review]

**Comment by jesopo** [comment]
I'll merge this when the tests pass, but I also think we should consider switching from env variables to command line arguments

**Pull request merged**